### DISCUSSION TITLE
KAN intuition for setting the hidden layers

### DISCUSSION THREAD

**Discussion by marcopeix** [Blealtan/efficient-kan#33] [General]
Hello,

Is there a rule of thumb or intuition for setting the layers_hidden parameter? I'm using it for time series, and I use [input_size, 10, horizon]. The 10 is arbitrary, and taken from the MNIST example, but do you have a suggestion on setting these for best performance?

**Comment by hoangthangta** [comment]
So far, I know that the number of hidden neurons can affect performance. In the MNIST example, I tried with the size 128, 256, and 512 instead of 64. However, there is a limit. Not sure with other parameters.

model = KAN([28 * 28, 256, 10])
